airbyte-api-server/src/main/resources/application.yml

```diff
@@ -0,0 +1,24 @@
+micronaut:
+  application:
+    name: airbyteApiServer
+#  http:
+#    services:
+#      config-api:
+#        url: "http://localhost:8000"
+#        read-timeout: 1m
+#        logger-name: config-api
+#        log-level: TRACE
+#        access-logger:
+#          enabled: true # Enables the access logger
+  server:
+    netty:
+      access-logger:
+        enabled: true # Enables the access logger
+airbyte:
+  configServerUrl: "localhost:8080"
```

Comment:
Convention in Micronaut is to use lowercase, hyphen-separate names.  It will automatically normalize camel/kebab case into that, so this still works.  However, to be consistent we should use lowercase with hyphens.